Erp connector (#418)
akeneo/pim-api-docs: content/guides/erp-connection/introduction.md

# Introduction
![Product illustration](../../img/illustrations/illus--Product.svg)

> You want to connect your ERP solution to your PIM?
> You want to implement an ERP connector and you don't know where to start?

You landed in the right place!

## Goal of this guide

In the following guide, we are going to explain how to properly connect an ERP to our PIM, thanks to our wonderful API.

We're here to help you find your way around, understand how your ERP solution will interact with our PIM and guide you through the steps to reach that goal.

> ## Building a generic solution?

Reviewer: ```suggestion
## Building a one-size-fits-all solution?
```

> If you are using this guide with the goal of building a **generic** connector for our Marketplace, you should know that each ERP solution has different capabilities and above all, each project has its own specificities due to the product modeling done in the ERP.

Reviewer: ```suggestion
If you are using this guide in order to build a **one-size-fits-all** connector for our Marketplace, you should know that each ERP solution has different capabilities and above all, each project has its own specificities due to the product modeling specific to each ERP.
```

It is therefore very complex to build a connector that fits all end-users' needs...

> That's why, from our experience, this guide will only guide you on the main use-cases and classical architectures.

Reviewer: ```suggestion
That's why, from our experience, this guide will only address the most common use-cases and classical architectures.
```

## ERP vs PIM product data

Before starting this guide, you need to understand that Akeneo PIM should only manage **"cold" product information**. It means that Akeneo PIM only owns marketing product information that needs to be checked, organized, enriched. And “cold” product information means that this information does not change every day.

> Concerning **“hot product information”** (i.e. product information that changes frequently: as prices or stock data), this information should be directly transmitted from your ERP solution to the eCommerce solution without passing through Akeneo PIM.

Reviewer: ```suggestion
Concerning **“hot product information”** (i.e. product information that changes frequently: such as prices or stock data), this information should be directly transmitted from your ERP solution to the eCommerce solution without going through Akeneo PIM.
```

> For example, **up-to-date price and stock information** of a product should never be stored in Akeneo PIM and should be transmitted directly from the ERP into the eCommerce solution.

Reviewer: ```suggestion
For example, the **up-to-date price and stock information** of a given product should never be stored in Akeneo PIM and should be transmitted directly from the ERP to the eCommerce solution.
```

:::info

> For price information, Akeneo PIM has the capability of storing product price information, but only for a **“reference price”** and not an **“up-to-date”** price. This **reference price** is only used by PIM users for information purposes and is therefore normally never exported from the PIM to a final solution.

Reviewer: ```suggestion
As far as prices are concerned, Akeneo PIM can store product price information, but only for  **“reference prices”** and not **“up-to-date”** ones. This **reference price** may exist to help PIM users in their daily work, and therefore is not meant to be exported.  
```

:::

> Ok... now that this product data dispatch from your ERP solution is crystal clear, let's start! 🚀

Reviewer: ```suggestion
Now that this product data dispatch from your ERP solution is crystal clear, let's start! 🚀
```
